cpmpy/solvers/solver_interface.py

```diff
@@ -199,9 +199,17 @@ def solveAll(self, display=None, time_limit=None, solution_limit=None, **kwargs)
 
             Returns: number of solutions found
         """
```

Comment:
`hasObjective(..)` could be implemented here as default ?

Reply:
Done :)